bin/rpmgrill

```diff
@@ -57,6 +57,7 @@ OPTIONS:
 
   --list-plugins     list available plugins
   --disable=[LIST]   disable one or more plugins, e.g.: --disable=virus,buildlog
+  --blacklist=[FILE] do not report fails in tests listed in blacklist file
```

Comment:
Ok I will rewrite it to use yaml format but it's still a blacklist not a whitelist and for CI we would rather have a blacklist which states what tests are disabled.